CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
-## Unreleased
+## 0.6.0
+
+### Vision
+- Added MapboxAccounts dependency
+- Added methods for setting segmentation and detection FPS to `VisionManagerInterface`
```

Comment:
This is the type of change that doesn't make a difference for the `MapboxVision` users. Consider removing this line.

Reply:
ah, sure, will remove

Reply:
done